test_fms/diag_integral/test_diag_integral.F90

```diff
@@ -175,12 +175,13 @@ end subroutine test_sum_diag_integral_field
   !-------------------------------------
   subroutine read_diag_integral_file
 
-    character(*), parameter :: di_file='diag_integral.out'
-    integer :: iunit
+    character(17), parameter :: di_file='diag_integral.out'
+    integer, parameter  :: iunit=100
+
     character(100) :: cline1, cline2, cline3, cline4, cline5, clin6
 
     !> read in computed values
-    open(newunit=iunit, file=di_file)
+    open(unit=iunit,file=trim(di_file))
```

Comment:
This should be changed back to using newunit rather than a hardcoded unit number. It's typically a better practice and I think this would also break the test with cray since it reserves 100 for stdin.

Reply:
Andrew just needs to merge in the main branch - he made these changes before Jesse's newunit fix got merged in so now it shows as Andrew changing it back

Reply:
Yes, Lauren is correct. I'll fix it